src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -151,6 +153,12 @@ class SharedLinkSection extends React.Component<Props, State> {
                     onCopyError();
                 });
         }
+
+        if (sharedLink.url === null) {
+            if (this.toggleRef) {
+                this.toggleRef.focus();
```

Comment:
This will also only focus on the toggle component when the `autofocusSharedLink` prop is false https://github.com/box/box-ui-elements/blob/master/src/features/unified-share-modal/SharedLinkSection.js#L242. Is that at odds with the accessibility requirement?

Reply:
@swfree when `autofocusSharedLink = false` the requirement says that it should focus in the invite people input field.